hw/ip_templates/pinmux/data/pinmux.tpldesc.hjson

```diff
@@ -57,5 +57,11 @@
       type: "int"
       default: "8"
     }
+    {
+      name: "enable_usb_wakeup"
+      desc: "Enable USB wakeup"
+      type: "int"
```

Comment:
That will be changed to `bool` once https://github.com/lowRISC/opentitan/pull/25013 is merged.